ci/build-test.sh

```diff
@@ -7,13 +7,18 @@ cd "$(dirname "$0")/../"
 # Get shared CI and prepare unity
 ci/bootstrap.sh
 .shared-ci/scripts/prepare-unity.sh
+.shared-ci/scripts/prepare-unity-mobile.sh "$(pwd)/logs/PrepareUnityMobile.log"
```

Comment:
Should we execute this only for Windows?

Reply:
Yup, good point.

Reply:
we probably do want to run it for both mac and windows at some point. but seeing as we are very limited with the number of mac agents we have and mac is kind of rekt anyway. we can isolate it to windows for now

Reply:
decided to execute it for both as it is possible for only one platform to fail